README.md

```diff
@@ -1,20 +1,14 @@
-# RBIs
+# `sorbet-typed`
+
+> A central repository for sharing type defintions for Ruby gems
 
-Database of `.rbi` files used for typing Gems and other Ruby classes.
 Inspired by [definitelytyped.org](http://definitelytyped.org/).
 Used in conjunction with [Sorbet](https://github.com/stripe/sorbet).
 
 ## Installation
 
-`gem install rbis --source <private gem source>`
-
-This is not yet a public gem: Coinbase currently mirrors this repo internally and builds to an internal gem server.
-
-## Project .rbi files
-
-To list all the `.rbi` files available to the current project, simply install this gem and run `rbis`.
-
-This makes running `sorbet` much easier, as you can simply pass the output of `rbis` to `sorbet` rather than manually collecting the relevant `.rbi` files.
+Use the Sorbet gem. `srb init` will automatically put the applicable files from
+this repository in `./sorbet/rbi/sorbet-typed`.
```

Comment:
What about `fetch the applicable files from this repository into your project` so that we don't have to change this README when we make changes to implementation details of `srb init`?